koswat/strategies/strategy_reinforcement_type.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from typing import Type
+
+from koswat.dike_reinforcements.reinforcement_profile.reinforcement_profile_protocol import (
+    ReinforcementProfileProtocol,
+)
+
+
+@dataclass
+class StrategyReinforcementType:
```

Comment:
Why would you change the name and the class location? The whole idea of this dataclass is to proxy costs per reinforcement type. Keep in mind doing these changes during parallel work brings conflicts to the other parties (me).
Revert.